rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/run/configurations/UnityAttachToEditorRunConfiguration.kt

```diff
@@ -92,6 +93,26 @@ class UnityAttachToEditorRunConfiguration(project: Project, factory: Configurati
         return null
     }
 
+    override fun checkRunnerSettings(
```

Comment:
Does this method have the same result as `checkSettingsBeforeRun`? I seem to remember that `checkSettingsBeforeRun` would show the editor form, so we can choose an editor to attach to, while `checkRunnerSettings` throws an exception and shows it as a balloon notification. Have I remembered that right?

Reply:
they both show the editor form for me.